README.md

```diff
@@ -2,7 +2,127 @@
 
 # Metamath set.mm repository
 
-This is a set of Metamath source files (databases) for logic and set theory.
+This is a collection of databases that specify mathematical axioms and
+rigorous formal proofs of various mathematical theorems
+derived from those axioms.
+This collection is specified and derived using the Metamath system.
+A "database" in Metamath parlance is a collection of one or more text files.
 
```

Comment:
Maybe to point to the *.mm-files which "are" the databases" would be usefull, because there are many other files in this repository.

Reply:
Fair enough. I'll add hyperlinks from each database name in the list of databases. That's probably the easiest way to do that.